Build.proj

```diff
@@ -22,8 +21,8 @@
   <Import Project="$(RepositoryEngineeringDir)restore\optimizationData.targets" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(EnableNgenOptimization)' == 'true'" />
 
   <Target Name="BuildLocalTasks"
-          BeforeTargets="Restore">
+          BeforeTargets="Build">
```

Comment:
@ViktorHofer, if we are consuming the default restore target from SDK, is the comment about `ProjectReferences` in packaging.stubs.targets still holds: https://github.com/dotnet/runtime/blob/fe33bed50573fa548e9daf00b90e016c401b6440/src/installer/packaging.stubs.targets#L3-L6
Maybe that file can be deleted? (if not, it could be inlined in `src/installer/Directory.Build.targets`)

Reply:
Spot on @am11. I thought about this couple of times but I honestly don't know if that is still required so I left it. I would like to get rid of the (host) pkgprojs and just invoke -pack on .proj files that do the packaging in a much easier way. These host proj files should also live next to the host's source code ideally. Haha there's so much to do in dotnet/runtime 😄